lib/client.dart

```diff
@@ -31,6 +31,13 @@ typedef void RoutePreLeaveEventHandler(RoutePreLeaveEvent event);
 typedef void RouteLeaveEventHandler(RouteLeaveEvent event);
 
 /**
+ * WindowClickHandler can be used as a hook into [Router] to
+ * modify behavior right after user clicks on an element, and
+ * before the URL in the browser changes.
+ */
+typedef void WindowClickHandler(Event e);
```

Comment:
Is this still used? Looks like the `clickHandler` optional args have been removed below.